test/verify/netlib.py

```diff
@@ -29,8 +29,16 @@ def setUp(self):
         # connections that might still be here from the time of
         # creating the image and we prevent NM from automatically
         # creating new connections.
-
-        m.execute("""nmcli -f UUID,DEVICE connection show | awk '$2 == "--" { print $1 }' | xargs -r nmcli con del""")
+        # if the command fails, try again
+        failures_allowed = 3
+        while True:
+            try:
+                m.execute("""nmcli -f UUID,DEVICE connection show | awk '$2 == "--" { print $1 }' | xargs -r nmcli con del""")
+                break
+            except:
```

Comment:
Nitpick: Bare `except:`s are rather dangerous -- this would even paper over things like `NameError` or `SyntaxError`. Better catch `subprocess.CalledProcessError` only?